.github/scripts/fetch_prs.mjs

```diff
@@ -12,72 +12,97 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+
 import { Octokit } from '@octokit/core';
 import axios from 'axios';
 import moment from 'moment';
 
 const owner = 'deckhouse';
 const repo = 'virtualization';
-const project = 'DVP'
-
+const project = 'DVP';
+const defaulLogin = 'NEED ATTENTION, reviewers are required';
```

Comment:
`t` is missing:
defaulLogin -> defaultLogin

Reply:
Yes, fixed